README.md

```diff
@@ -1,38 +1,41 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+This project name is Portfolio: add a contact form and finish the mobile version
```

Comment:
- [ ] [OPTIONAL] Greet job adding the project name, however, it's a good idea to add the name directly in the heading (# Project Name) instead of adding it this way, kindly consider changing `# Project Name` to your project name, (`# Portfolio: add a contact form and finish the mobile version`)